src/openrct2/cmdline/BenchSpriteSort.cpp

```diff
@@ -150,7 +150,7 @@ static void BM_paint_session_arrange(benchmark::State& state, const std::vector<
         state.PauseTiming();
         std::copy_n(local_s, std::size(sessions), sessions.begin());
         state.ResumeTiming();
-        PaintSessionArrange(&sessions[0].Session);
+        PaintSessionArrange(*(&sessions[0].Session));
```

Comment:
@ZehMatt Is this correct, or should it be 
```suggestion
        PaintSessionArrange(sessions[0].Session);
```
?

Reply:
You are correct about that.

Reply:
Which one is correct? The code in my suggestion?

Reply:
Yes, the first one is doing Dereference(PointerOf(Sessions[0].Session)) which makes no sense really.